tests/Mutator/Boolean/Yield_Test.php

```diff
@@ -27,13 +27,19 @@ public function provideMutationCases(): \Generator
             <<<'PHP'
 <?php
 
-(yield $a => $b);
+function test()
```

Comment:
Before this change, this mutator was failing with `PHP Fatal error:  The "yield" expression can only be used inside a function in /tmp/XXX`